lighthouse-extension/app/src/popup.js

```diff
@@ -49,6 +49,10 @@ function getLighthouseVersion() {
   return chrome.runtime.getManifest().version;
 }
 
+function getLighthouseRevision() {
```

Comment:
bikeshed: can we call this `getLighthouseCommitHash` or something like that (and update references to `REVISION` below)? I wasn't sure what "revision" was referring to until I got to where it was defined :)

`**Lighthouse Revision**` below could be `**Lighthouse Commit**`, for instance.

Just an idea, though. Open to other ways people refer to the exact commit